### PR TITLE
turn verbs into nouns for gallery homepage

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
cc @HansMuller

**Comment by sethladd** [comment]
closes https://github.com/flutter/flutter/issues/5681

File: examples/flutter_gallery/lib/gallery/item.dart [hunk 1]
```diff
@@ -80,13 +80,13 @@ final List<GalleryItem> kAllGalleryItems = <GalleryItem>[
...
     title: 'Chips',
-    subtitle: 'A label with an optional delete button and avatar',
+    subtitle: 'Label with an optional delete button and avatar',
     routeName: ChipDemo.routeName,
     buildRoute: (BuildContext context) => new ChipDemo()
   ),
   new GalleryItem(
     title: 'Date picker',
-    subtitle: 'Choose month, day, and year',
+    subtitle: 'Calendar date selection widget',
```
Comment (dragostis) [inline examples/flutter_gallery/lib/gallery/item.dart]: Maybe just 'Date selection widget'?

**Comment by dragostis** [comment]
LGTM.

**Comment by HansMuller** [comment]
This LGTM as well. 

The subtitles now all seem fill in the blank from: What is this thing? It is a [...]. We should keep that in mind when adding more subtitles.

**Pull request merged**